### PR TITLE
Added functionality to import transient data

### PR DESCRIPTION
**Pull request by zdanaceau**

### CODE REVIEW

File: FCWTAPI/TransientData.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace FCWTNET
+{
+    public class TransientData
+    {
+        public string Filepath { get; set; }
```
Comment (avcarr2) [inline FCWTAPI/TransientData.cs]: Change FilePath to readonly (getter setter to { get; } ).

File: FCWTAPI/TransientData.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        /// <summary>
+        /// Imports transient data into the TransientData class
+        /// </summary>
+        /// <param name="maxTimePoints">Maximum number of timepoints to read from the transient CSV file</param>
+        /// <exception cref="FormatException"></exception>
+        /// <exception cref="ArgumentException"></exception>
+        /// <exception cref="FileNotFoundException"></exception>
+        public void ImportTransientData(int maxTimePoints = int.MaxValue - 4)
+        {
+            if (Filepath.Substring(Filepath.Length - 4, 4) != ".csv")
```
Comment (avcarr2) [inline FCWTAPI/TransientData.cs]: Would recommend using methods from the System.IO methods for file input and output. This is a clever solution, but not ideal

File: FCWTAPI/TransientData.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                        LineCounter++;
+                    }
+
+                }
+            }
+            catch(FileNotFoundException)
+            {
+                throw new FileNotFoundException("Transient data file could not be found");
+            }
+            double[] dataArray = new double[dataList.Count];
```
Comment (avcarr2) [inline FCWTAPI/TransientData.cs]: Is there some a compiler warning or error preventing 
> Data = dataList.ToArray(); 
from working?

**Review by avcarr2 — CHANGES_REQUESTED** [review]
Some changes to make.

File: FCWTAPI/TransientData.cs (hunk 3 above)
Comment (zdanaceau) [reply FCWTAPI/TransientData.cs:80]: There is none, I just didn't know that existed until this morning.

Comment (zdanaceau) [reply FCWTAPI/TransientData.cs]: There is none.

**Pull request merged**